docs/upgrade.md

```diff
@@ -0,0 +1,93 @@
+- Spoke cluster is successfully deployed and configured using a `ProvisioningRequest`:  
+```bash
+$ oc get provisioningrequests.o2ims.provisioning.oran.org        
+NAME      AGE   PROVISIONSTATE   PROVISIONDETAILS
+cluster-name   37m   fulfilled        Provisioning request has completed successfully
+```
+- [OADP and LCA is installed on the spoke cluster. OADP backend is configured on the spoke cluster.](https://docs.openshift.com/container-platform/4.17/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-install-operators.html)
+```yaml
+# policytemplates/version_4.16.x/sno-ran-du/sno-ran-du-pg-v4-16-x-v1.yaml
+
+policies:
+- name: operator-subscriptions
+  manifests:
+    # LCA
+    - path: source-crs/LcaOperatorStatus.yaml
+    - path: source-crs/LcaSubscriptionNS.yaml
+    - path: source-crs/LcaSubscriptionOperGroup.yaml
+    - path: source-crs/LcaSubscription.yaml
+    # OADP
+    - path: source-crs/OadpOperatorStatus.yaml
+    - path: source-crs/OadpSubscriptionNS.yaml
+    - path: source-crs/OadpSubscriptionOperGroup.yaml
+    - path: source-crs/OadpSubscription.yaml
+- name: oadp-backend
+  manifests:
+    - path: source-crs/OadpSecret.yaml
+      patches:
+        - data: <FILL>
+    - path: source-crs/OadpDataProtectionApplication.yaml
+      patches:
+        - spec:
+            backupLocations:
+            <FILL>
+```
+### steps:
+- Create new clustertemplates and policytemplates directories for the new release according to sample git-setup.
+- Update release version in new clustertemplates files
+```yaml
+# clustertemplates/version_4.16.y/sno-ran-du/clusterinstance-defaults-v1.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: clusterinstance-defaults-v1
+  namespace: sno-ran-du-v4-16-y
+data:
+  clusterinstance-defaults: |
+    clusterImageSetNameRef: "4.16.y"
+---
+# clustertemplates/version_4.16.y/sno-ran-du/sno-ran-du-v4-16-y-1.yaml
+kind: ClusterTemplate
+metadata:
+  name: sno-ran-du.v4-16-y-1
+  namespace: sno-ran-du-v4-16-y
+spec:
+  name: sno-ran-du
+  version: v4-16-y-1
+  release: 4.16.y
+---
+# clustertemplates/version_4.16.y/sno-ran-du/upgrade-defaults-v1.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: upgrade-defaults-v1
+  namespace: sno-ran-du-v4-16-y
+data:
+  ibgu: |
+    ibuSpec:
+      seedImageRef:
+        image: "quay.io/seed-repo/seed-image:4.16.y"
+        version: "4.16.y"
+      oadpContent:
+        - name: sno-ran-du-ibu-platform-backup-v4-16-y-1
+          namespace: openshift-adp
+```
+- Sync ztp repo to the hub cluster.
+- Patch `ProvisioningRequest.spec.templateName` and `ProvisioningRequest.spec.templateVersion` to point to the new `ClusterTemplate`. This will trigger the image based upgrade.
```

Comment:
Should we mention that the templateParameters may also require updates if there are changes to the templateParameterSchema in the new ClusterTemplate version?